database/seed.js

```diff
@@ -10,44 +10,50 @@ const randomIntFromInterval = (min, max) =>// min and max included
   return Math.floor(Math.random()*(max-min+1)+min);
 }
 
-const generateReviews = (numberOfWatches) => {
-	const firstWatchId = 101;
-  let nextReviewId = 1001; // auto-increment
-  let reviewArray = [];
-
-  for (let watchId = firstWatchId; watchId <= numberOfWatches + firstWatchId; watchId += 1) {
-    let reviewsForThisWatch = randomIntFromInterval(4, 100);
-    for (let review = 1; review <= reviewsForThisWatch; review += 1) {
-      const record = {};
-      record._id = nextReviewId;
-      record.w_id = watchId;
-      record.reviewer = faker.name.findName();
-      record.stars = faker.random.number({
-        'min': 1,
-        'max': 5
-      });
-      record.date_posted = faker.date.past();
-      record.review_header = faker.lorem.words();
-      record.review_body = faker.lorem.paragraphs();
-      record.upvotes = faker.random.number(100);
-      record.downvotes = faker.random.number(100);
-      nextReviewId++;
-
-    	reviewArray.push(record);
-    }
-  }
-  return reviewArray;
+const firstWatchId = 101;
```

Comment:
Per our conversation, might want to consider starting the csv at index 100 for the display page.